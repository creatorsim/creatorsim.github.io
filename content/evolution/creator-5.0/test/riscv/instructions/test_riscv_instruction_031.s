
#
# Creator (https://creatorsim.github.io/creator/)
#
 
.data
    a: .double 34.544
    b: .double 11.443
    c: .dword  0x7544FFB
    
    d: .zero 24
    
.text
main:
    la x5, a
    la x6, b
    la x7, c
    la x8, d
    
    fld   f0,  0(x5)
    fld   f2,  0(x6)
    fld   f10, 0(x7)
    
    fsd  f0,  0(x8)
    addi x8, x8, 8
    fsd  f2,  0(x8)
    addi x8, x8, 8
    fsd  f10, 0(x8)

    li a7, 10
    ecall

