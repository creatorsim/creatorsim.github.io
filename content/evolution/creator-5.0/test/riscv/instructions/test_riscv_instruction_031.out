
cr[PC]:0x48; ir[x5,t0]:0x200000; ir[x6,t1]:0x200008; ir[x7,t2]:0x200010; ir[x8,fp,s0]:0x200028; ir[x17,a7]:0xa; fpr[f0,ft0]:0x404145A1CAC08312; fpr[f2,ft2]:0x4026E2D0E5604189; fpr[f10,fa0]:0x0000000007544FFB; memory[0x20002f]:0xFB; memory[0x20002e]:0x4F; memory[0x20002d]:0x54; memory[0x20002c]:0x07; memory[0x200027]:0x89; memory[0x200026]:0x41; memory[0x200025]:0x60; memory[0x200024]:0xE5; memory[0x200023]:0xD0; memory[0x200022]:0xE2; memory[0x200021]:0x26; memory[0x200020]:0x40; memory[0x20001f]:0x12; memory[0x20001e]:0x83; memory[0x20001d]:0xC0; memory[0x20001c]:0xCA; memory[0x20001b]:0xA1; memory[0x20001a]:0x45; memory[0x200019]:0x41; memory[0x200018]:0x40; keyboard[0x0]:''; display[0x0]:''; 

